dist/stat_data.py

```diff
@@ -282,8 +282,13 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_insert', 'cache overflow table insert calls'),
     CacheStat('cache_lookaside_ondisk', 'cache overflow table on-disk size', 'no_clear,no_scale,size'),
     CacheStat('cache_lookaside_ondisk_max', 'cache overflow table max on-disk size', 'no_clear,no_scale,size'),
+    CacheStat('cache_lookaside_read', 'cache overflow table update reads'),
+    CacheStat('cache_lookaside_read_miss', 'cache overflow table update reads missed'),
```

Comment:
This is tracking how often we call the `__wt_find_lookaside_upd` and don't get anything useful. Even though we're searching the las relatively efficiently it will still be expensive.